content/client-lib-development-guide/features.textile

```diff
@@ -81,35 +81,37 @@ h3(#restclient). RestClient
 
 h3(#rest-auth). Auth
 
-* @(RSA1)@ Basic Auth connects over HTTPS by default.  If HTTP and Basic Auth are used, an exception will be raised
+* @(RSA1)@ "Basic Auth":https://en.wikipedia.org/wiki/Basic_access_authentication connects over HTTPS by default.  If HTTP and Basic Auth are used, an exception will be raised
```

Comment:
As above, I just added a basic auth link, but will find all instances now and address in one.